model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city_name
+        stability: experimental
+        type: string
+        brief: >
+          City name.
+        examples: [ 'Montreal', 'Berlin' ]
+      - id: continent_code
+        stability: experimental
+        brief: >
+          Two-letter code representing continent’s name.
+        type:
+          members:
+            - id: 'af'
+              stability: experimental
+              value: 'AF'
+              brief: 'Africa'
+            - id: 'an'
+              stability: experimental
+              value: 'AN'
+              brief: 'Antarctica'
+            - id: 'as'
+              stability: experimental
+              value: 'AS'
+              brief: 'Asia'
+            - id: 'eu'
+              stability: experimental
+              value: 'EU'
+              brief: 'Europe'
+            - id: 'na'
+              stability: experimental
+              value: 'NA'
+              brief: 'North America'
+            - id: 'oc'
+              stability: experimental
+              value: 'OC'
+              brief: 'Oceania'
+            - id: 'sa'
+              stability: experimental
+              value: 'SA'
+              brief: 'South America'
+      - id: continent_name
+        stability: experimental
+        type: string
+        brief: >
+          Name of the continent.
+        examples: [ 'North America', 'Europe' ]
+      - id: country_iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://en.wikipedia.org/wiki/ISO_3166-1#Codes)).
+        examples: [ 'CA' ]
+      - id: country_name
+        stability: experimental
+        type: string
+        brief: >
+          Country name.
+        examples: [ 'Canada' ]
+      - id: location.lon
+        stability: experimental
+        type: double
+        brief: >
+          Longitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ -73.614830 ]
+      - id: location.lat
+        stability: experimental
+        type: double
+        brief: >
+          Latitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ 45.505918 ]
```

Comment:
I am wondering, whether the reference to WGS84 should be replaced by an [EPSG identifier as an SRID](https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset). But this might be a step to far into [georeferencing](https://en.wikipedia.org/wiki/Georeferencing), which might not be required at this point in time. The convention could be refined and extended to support other reference systems, that require more fields than just a longitude/lattitude pair.

Reply:
I'd recommend to add that later when we have a need / use case for that specifically.